download-aspera/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:18.04
+
+LABEL org.opencontainers.image.source https://github.com/edsu7/argo-data-submission
```

Comment:
use `icgc-argo` instead of personal repo: `https://github.com/icgc-argo/argo-data-submission` ?